docs/userguide/index.rst

```diff
@@ -10,6 +10,15 @@ packages.
 Packages built and distributed using ``setuptools`` look to the user like
 ordinary Python packages based on the ``distutils``.
 
+Transition to PEP517
+====================
+
+Since setuptools no longer serves as the default build tool, one must explicitly
+opt in (by providing a ``pyproject.toml`` file) to use this library. The user
+facing part is provided by `PEP517 <https://pypi.org/project/pep517/>`_ and
```

Comment:
I'm pretty sure that this shouldn't point users to a specific implementation. There's plenty, including pip. I'd rather point to the PEP page using
```rst
:pep:`517`
```

Besides, the CLI present in the pep517 package has always been unstable and experimental, and it is being deprecated in favor of another project called `build` that is about to be accepted under the roof of PyPA: https://mail.python.org/archives/list/pypa-committers@python.org/thread/LNOO7Q2RW55BVYCJK2LP4DEIMHC5D44F/.